### PR TITLE
Receive: add labels + depth when clicking

### LINKED ISSUE #2518: Receive page: buttons have no depth when clicking
Orange buttons in GUI are responsive to clicking:

But this is not seen in Orange buttons with icons of Receive page:

### PR DESCRIPTION
**Pull request by rating89us**
Closes #2518

* Buttons in rows
* Adding labels (while tooltips are not implemented yet)
* Changing icons to FontAwesome (see issue #2518)

Before:
image

After:
buttons

### CODE REVIEW

File: pages/Receive.qml [hunk 1]
```diff
@@ -291,26 +291,37 @@ Rectangle {
             }
 
             RowLayout {
-                spacing: parent.spacing
-
                 MoneroComponents.StandardButton {
-                    rightIcon: "qrc:///images/download-white.png"
+                    Layout.preferredWidth: 220
+                    small: true
+                    text: FontAwesome.save + qsTr("  Save as image") + translationManager.emptyString
```
Comment (selsta) [inline pages/Receive.qml]:
```suggestion
                    text: FontAwesome.save + "  %1".arg(qsTr("Save as image")) + translationManager.emptyString
```

The spaces are not interesting for translations. Better to split it up. Please also change the other.

File: pages/Receive.qml [hunk 2]
```diff
@@ -321,7 +332,7 @@ Rectangle {
                 }
             }
         }
-
+
```
Comment (selsta) [inline pages/Receive.qml]: No unrelated whitespace changes please.

File: pages/Receive.qml [hunk 3]
```diff
@@ -291,26 +291,37 @@ Rectangle {
...
                     onClicked: qrFileDialog.open()
                 }
+            }
 
+            RowLayout {
                 MoneroComponents.StandardButton {
-                    rightIcon: "qrc:///images/external-link-white.png"
+                    Layout.preferredWidth: 220
+                    small: true
+                    text: FontAwesome.clipboard + "  %1".arg(qsTr("Copy to cliboard")) + translationManager.emptyString
```
Comment (selsta) [inline pages/Receive.qml]:
```suggestion
                    text: FontAwesome.clipboard + "  %1".arg(qsTr("Copy to clipboard")) + translationManager.emptyString
```

Small typo

File: pages/Receive.qml [hunk 4]
```diff
@@ -291,26 +291,37 @@ Rectangle {
...
+                    Layout.preferredWidth: 220
+                    small: true
+                    text: FontAwesome.save + "  %1".arg(qsTr("Save as image")) + translationManager.emptyString
+                    label.font.family: FontAwesome.fontFamily
+                    fontSize: 13
                     onClicked: qrFileDialog.open()
                 }
+            }
 
+            RowLayout {
```
Comment (xiphon) [inline pages/Receive.qml]:
```suggestion
```

Comment (xiphon) [reply pages/Receive.qml]: And other RowLayouts as well

File: pages/Receive.qml [hunk 5]
```diff
@@ -290,34 +290,39 @@ Rectangle {
                 }
             }
 
-            RowLayout {
-                spacing: parent.spacing
-
-                MoneroComponents.StandardButton {
-                    rightIcon: "qrc:///images/download-white.png"
```
Comment (xiphon) [inline pages/Receive.qml]: Please remove unused images from the repo and `qml.qrc`.

File: pages/Receive.qml [hunk 6]
```diff
@@ -290,34 +290,39 @@ Rectangle {
...
+                Layout.preferredWidth: 220
+                small: true
+                text: FontAwesome.save + "  %1".arg(qsTr("Save as image")) + translationManager.emptyString
+                label.font.family: FontAwesome.fontFamily
+                fontSize: 13
+                onClicked: qrFileDialog.open()
+            }
 
-                MoneroComponents.StandardButton {
-                    rightIcon: "qrc:///images/external-link-white.png"
```
Comment (selsta) [inline pages/Receive.qml]: can also be removed

Comment (selsta) [inline pages/Receive.qml]: can also be removed
